src/modules/site/base/views/about.py

```diff
@@ -40,7 +40,7 @@ def about():
   ''' About us Page - Gives an overview of CeNDR '''
   title = "About CeNDR"
   disable_parent_breadcrumb = True
-  isotypes = get_isotypes(known_origin=True)
+  isotypes = get_isotypes(known_origin=True, species='c_elegans')

```

Comment:
Avoid hardcoded strings -- see previous comment on creating a class/enum to represent the species.  (eg: models.TaskStatus).

Reply:
Removed in 1df7d83b8c2c915cccfd8fd9ad1b296727b39c5b.  See discussion at https://github.com/AndersenLab/CAENDR/pull/145#discussion_r1132918582.